### PR TITLE
Delete ethers lib

### PR DESCRIPTION
**Pull request by antonina-cherednichenko**
I realized that I can delete ethers and change a few usages inside Enconding.ts to web3 analogs.

### CODE REVIEW

File: tests/Utils/Ethereum.js [hunk 1]
```diff
@@ -59,7 +59,8 @@ function getContractDefaults() {
 }
 
 function keccak256(values) {
-  return ethers.utils.keccak256(values);
+  // alternatively use `return ethers.utils.keccak256(values)`
+  return web3.utils.soliditySha3(values);
```
Comment (jflatow) [inline tests/Utils/Ethereum.js:63]: Isn't there also `web3.utils.keccak256`?

Comment (jflatow) [reply tests/Utils/Ethereum.js:63]: Ah missed the comment, yes that seems better?

Comment (antonina-cherednichenko) [reply tests/Utils/Ethereum.js:63]: good catch, I am not sure what is the right thing here. I can use https://web3js.readthedocs.io/en/v1.2.7/web3-utils.html#sha3 as an exact replacement. 

Also, ethers library has `solidityKeccak256`

**Comment by hayesgm** [comment]
FWIW, isn't Ethers a dep of Web3, so isn't it being included one way or the other?

**Comment by antonina-cherednichenko** [comment]
> FWIW, isn't Ethers a dep of Web3, so isn't it being included one way or the other?

yes, it is. I thought that it would make sense to not use it explicitly because other usages are gone and the remaining 2 are easily replaceable to web3 analogs. This pr is optional if you think it makes sense to keep `ethers` lib.

**Pull request closed without merging**